tardis/energy_input/gamma_packet_loop.py

```diff
@@ -103,21 +101,20 @@ def gamma_packet_loop(
     escaped_packets = 0
     scattered_packets = 0
     packet_count = len(packets)
+    # Logging does not work with numba. Using print instead.
     print("Entering gamma ray loop for " + str(packet_count) + " packets")
```

Comment:
I would much prefer using logging, and just doing it right above the function call rather than inside the function.

Reply:
Yep now this is moving from being in-development, we should formalize some of these logs (which has already been done in some of the other major functions)